### ISSUE TITLE
External links should open in a new tab by default

### ISSUE THREAD

**Issue by aruizca** [aruizca/jingo#11]

**Comment by matthewandrews** [comment]
Agree this is reasonable so long as we have no staff using screenreaders, but not appropriate for a public wiki (accessibility), so it should be made configurable in case it ever gets merged back to the claudioc version.
